README.md

```diff
@@ -170,12 +170,42 @@ To start the trento agent:
 > recommendations on cluster settings and state for many HA components. New rules
 > can be created and tuned to adjust to different requirements.
 
+## Trento Runner
+
+Trento Runner is responsible for running the health checks. It is based on [Ansible](https://docs.ansible.com/ansible/latest/index.html) and [ARA](https://ara.recordsansible.org/).
+This component can be executed in the same machine as the Web UI, but it is not mandatory, it can be executed in any other machine that has network access to the agents.
+
+In order to start it, some packages must be installed and started. Here a quick go through:
+```shell
+# Install ansible. Tested with version 2.11.2
+pip install ansible
+# Install ARA
+pip install --user ansible "ara[server]"
+# Setup ARA database
```

Comment:
Is installing ansible again in the user directory needed?

Reply:
Let me check this. I followed the ARA installation guides, but maybe this repeats the installation twice.
What I thought it does is to install ara under the ansible user, which is something required by it.
I will double check

Reply:
Double checked and you were right @fabriziosestito , the installation is redundant. The source of the documentation by the way: https://ara.readthedocs.io/en/latest/getting-started.html